package.json

```diff
@@ -68,5 +71,23 @@
   "bugs": {
     "url": "https://github.com/jembi/openhim-console/issues"
   },
-  "homepage": "http://openhim.org"
+  "homepage": "http://openhim.org",
+  "standard": {
+    "globals": [
```

Comment:
@tumijacob It would be better to specify the browser and mocha environments instead of so many globals. See https://eslint.org/docs/user-guide/configuring.html#specifying-environments.